src/OpenAL/OpenToolkit.OpenAL/AudioContext.cs

```diff
@@ -404,13 +399,13 @@ IEnumerable<int> attributes
             if (!string.IsNullOrEmpty(device))
             {
                 _deviceName = device;
-                Device = ContextAPI.OpenDevice(device); // try to open device by name
+                Device = (Device*)ALC.OpenDevice(device); // try to open device by name
```

Comment:
`_deviceName` and `device` are a little misleading here, naming wise.

deviceName indicates a string, device indicates an AlcDevice or so.

Maybe consider renaming `device` to `deviceName`

Reply:
This file has been deleted in the latest.